migration.go

```diff
@@ -27,6 +27,10 @@ type Migration struct {
 	DownFn   func(*sql.Tx) error // Down go migration function
 }
 
+type TemplateParams struct {
+	Version, Name string
```

Comment:
We don't really need this struct, just pass two arguments to writeTemplateToFile() func